### PR TITLE
Add table column commands

### LINKED ISSUE #24: Add commands to move table columns, add new column

### LINKED ISSUE #25: Emacs commands should return the new line/col number

### PR DESCRIPTION
**Pull request by filipelbc**
Closes #24 

Closes #25

### CODE REVIEW

File: ftplugin/org.vim [hunk 1]
```diff
@@ -104,17 +120,24 @@ function! OrgExportToHTML()
...
+        OrgEchoError(l:out)
     endif
 endfunction
 
 function! OrgMakeProgn(prog)
-    if type(a:prog) == v:t_list
-        let l:prog = join(a:prog, ' ')
-    else
-        let l:prog = a:prog
+    " Support the different kinds of way to define a program (string, list,
```
Comment (jzsampaio) [inline ftplugin/org.vim:128]: It is not clear to me how this is used. Would you provide some context on what 'program' you're talking about?

Comment (jzsampaio) [reply ftplugin/org.vim:128]: Is this the org program that updates tables and stuff?

File: ftplugin/org.vim [hunk 2]
```diff
@@ -26,18 +26,34 @@ endif
...
+            \   },
+            \   'MoveColLeft': {
+            \       'prog': '(org-table-move-column-left)',
+            \       'cursor': 'col',
+            \   },
+            \   'DelCol': {
+            \       'prog': '(org-table-delete-column)',
+            \       'cursor': 'col',
+            \   },
+            \   'AddCol': {
```
Comment (jzsampaio) [inline ftplugin/org.vim:53]: Column is always added to the left. Is there a add left/right?

Comment (filipelbc) [reply ftplugin/org.vim:53]: I did not find in the orgmode documentation.

File: ftplugin/org.vim (hunk 1 above)
Comment (filipelbc) [reply ftplugin/org.vim:128]: This is going to be refactored in #31, then I should document it better. In any case, I've opened #32 too.

**Pull request merged**